src/callbacks_step/analysis_surface_2d.jl

```diff
@@ -0,0 +1,113 @@
+
+struct AnalysisSurfaceIntegral{Indices, Variable}
+    indices::Indices
+    variable::Variable
+end
+
+struct ForceState{RealT <: Real}
+    Ψl::Tuple{RealT, RealT}
+    rhoinf::RealT
+    uinf::RealT
+    linf::RealT
+end
+
+# TODO - This should be a struct in ForceState
+struct FreeStreamVariables{RealT <: Real}
+    rhoinf::RealT
+    uinf::RealT
+    linf::RealT
+end
+
+struct LiftForcePressure{RealT <: Real}
+    force_state::ForceState{RealT}
+end
+
+struct DragForcePressure{RealT <: Real}
+    force_state::ForceState{RealT}
+end
+
+function LiftForcePressure(aoa::Real, rhoinf::Real, uinf::Real, linf::Real)
+    # Ψl is the normal unit vector to the freestream direction
+    Ψl = (-sin(aoa), cos(aoa))
+    force_state = ForceState(Ψl, rhoinf, uinf, linf)
+    return LiftForcePressure(force_state)
+end
+
+function DragForcePressure(aoa::Real, rhoinf::Real, uinf::Real, linf::Real)
+    # Ψd is the unit vector parallel to the freestream direction
+    Ψd = (cos(aoa), sin(aoa))
+    return DragForcePressure(ForceState(Ψd, rhoinf, uinf, linf))
+end
+
+function (lift_force::LiftForcePressure)(u, normal_direction, equations)
+    p = pressure(u, equations)
+    @unpack Ψl, rhoinf, uinf, linf = lift_force.force_state
+    n = dot(normal_direction, Ψl) / norm(normal_direction)
+    return p * n / (0.5 * rhoinf * uinf^2 * linf)
+end
+
+function (drag_force::DragForcePressure)(u, normal_direction, equations)
+    p = pressure(u, equations)
+    @unpack Ψl, rhoinf, uinf, linf = drag_force.force_state
+    n = dot(normal_direction, Ψl) / norm(normal_direction)
+    return p * n / (0.5 * rhoinf * uinf^2 * linf)
+end
```

Comment:
These guys essentially compute the lift/drag coefficient, i.e., $C_L, C_D$ (see [Wikipedia](https://en.wikipedia.org/wiki/Lift_(force)#Quantifying_lift)). In that case, we should consider re-labeling.

Reply:
That is certainly a point for discussion - for the moment we should maybe just relabel this accordingly to avoid confusion for other people.

Reply:
Or just avoid the normalization as this is less work.

Reply:
In a perfect world, it would be great if by default the functions would return the usually expected quantity, but allow one to easily switch to a different output. For example, one could add an extra argument during the callback setup with a conversion function that by default computes lift and drag coefficients, but if one specifies, e.g., `id` (the identity), one could also get the underlying forces or pressure values.